crates/task-impls/src/da.rs

```diff
@@ -163,20 +158,16 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, V: Versions> DaTaskState<TYP
                 )
                 .await;
 
-                if !self.da_membership.has_stake(&self.public_key) {
-                    debug!(
+                ensure!(
+                    self.da_membership.has_stake(&self.public_key),
+                    format!(
                         "We were not chosen for consensus committee on {:?}",
                         self.cur_view
-                    );
-                    return None;
-                }
-                if let Err(e) = self.storage.write().await.append_da(proposal).await {
-                    error!(
-                        "Failed to store DA Proposal with error {:?}, aborting vote",
-                        e
-                    );
-                    return None;
-                }
+                    )
+                );
+
+                self.storage.write().await.append_da(proposal).await?;
```

Comment:
error -> info level

Reply:
all storage methods should now log at error!